### PR TITLE
Added nxos_lldp_interfaces resource module

### PR DESCRIPTION
**Pull request by adharshsrivatsr**
##### SUMMARY
Added nxos_lldp_interfaces collections resource module 

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME
modules/network/nxos/nxos_lldp_interfaces.py

### CODE REVIEW

File: lib/ansible/module_utils/network/nxos/argspec/lldp_interfaces/lldp_interfaces.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                        'management_address': {
+                            'type': 'str'
+                        },
+                        'vlan': {
+                            'type': 'int'
+                        }
+                    },
+                    'type': 'dict'
+                },
+                'transmit': {
```
Comment (trishnaguha) [inline lib/ansible/module_utils/network/nxos/argspec/lldp_interfaces/lldp_interfaces.py:59]: transmit parameter should be intended under config?

**Review by trishnaguha — CHANGES_REQUESTED** [review]
Looks like you deleted lacp_interfaces file. The pep8 failures need to be fixed.

File: lib/ansible/module_utils/network/nxos/argspec/lldp_interfaces/lldp_interfaces.py (hunk 1 above)
Comment (adharshsrivatsr) [reply lib/ansible/module_utils/network/nxos/argspec/lldp_interfaces/lldp_interfaces.py:59]: Yes, it is another parameter like receive.

**Comment by adharshsrivatsr** [comment]
recheck

**Comment by adharshsrivatsr** [comment]
recheck

**Pull request closed without merging**